docs/integrations/builtin/credentials/httprequest.md

```diff
@@ -14,76 +14,164 @@ You can use these credentials to authenticate the following nodes:
 
 You must use the authentication method required by the app or service you want to query.
 
-### Predefined credential types
+## Supported authentication methods
```

Comment:
If I read 'supported auth type' I expect the content that's in generic (all the different HTTP auth types) I would rename this something like 'Generic and predefined credentials', and include a sentence or two explaining what they are, perhaps with link to other relevant docs?